package.json

```diff
@@ -81,46 +102,41 @@
     "babel-cli": "6.26.0",
     "babel-core": "6.26.0",
     "babel-eslint": "8.0.2",
+    "babel-jest": "21.2.0",
     "babel-loader": "7.1.2",
+    "babel-plugin-dynamic-import-node": "1.2.0",
     "babel-plugin-import": "1.6.2",
-    "babel-plugin-istanbul": "4.1.5",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.26.0",
     "babel-plugin-transform-import-extension-jsx-to-js": "0.1.0",
     "babel-polyfill": "6.26.0",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
-    "coveralls": "3.0.0",
+    "canvas-prebuilt": "1.6.5-prerelease.1",
     "css-loader": "0.28.7",
     "enzyme": "3.1.1",
     "enzyme-adapter-react-16": "1.0.4",
     "eslint": "4.11.0",
     "eslint-plugin-html": "4.0.0",
     "eslint-plugin-markdown": "1.0.0-beta.7",
     "eslint-plugin-react": "7.4.0",
-    "expect.js": "0.3.1",
     "file-loader": "1.1.5",
     "image-webpack-loader": "3.4.2",
     "ink-docstrap": "1.3.1",
+    "jest": "21.2.1",
     "jsdoc": "3.5.5",
     "jsdoc-babel": "0.3.0",
-    "karma": "1.7.1",
-    "karma-chrome-launcher": "2.2.0",
-    "karma-cli": "1.0.1",
-    "karma-coverage": "1.1.1",
-    "karma-firefox-launcher": "1.0.1",
-    "karma-mocha": "1.3.0",
-    "karma-mocha-reporter": "2.2.5",
-    "karma-sourcemap-loader": "0.3.7",
-    "karma-webpack": "2.0.6",
     "less": "2.7.3",
     "less-loader": "4.0.5",
     "metalsmith": "2.3.0",
     "metalsmith-collections": "0.9.0",
     "metalsmith-layouts": "1.8.1",
     "metalsmith-markdown": "0.2.1",
     "minami": "1.2.3",
-    "mocha": "4.0.1",
+    "ol": "4.5.0",
+    "react": "16.1.1",
```

Comment:
OpenLayers and React are already peer dependendies, aren't they?

Reply:
They where added intentionally -> see PR description.

Reply:
Oh, I should have read description more carefully. :see_no_evil:  Sorry for the noise

Reply:
I don't like it having them included as dev- and peerDependencies. Let's just create `postinstall` script like this (inside package.json):

```json
{
  "postinstall": "npm i --no-save ol@4 react@16"
}
```

Reply:
Whatever @dnlkoch suggests.